pkg/matrix/matrix.go

```diff
@@ -287,3 +287,15 @@ func CreateMetric(key string, data *Matrix) error {
 	}
 	return nil
 }
+
+func CopyInstances(source *Matrix, dest *Matrix) {
```

Comment:
are these needed? Can we use the existing Clone method instead?
`matrix.Clone(false, true, true)` the fixup the uuid, object, and identifier?

Reply:
matrix.Clone method would work with changing of uuid, object and identifier.